packages/xo-server-sdn-controller/src/index.js

```diff
@@ -607,6 +779,57 @@ class SDNController extends EventEmitter {
 
   // ---------------------------------------------------------------------------
 
+  async _electNewPoolCenter(crossPoolNetwork) {
```

Comment:
So that's what will elect the main pool and `_electNewCenter` will elect the main host within a given pool, right? I think this needs to be super clear in both function names.

Reply:
I agree, `poolNetwork` VS `crossPoolNetwork` and `starCenter` VS `poolCenter` is REALLY confusing.

I'm trying to find better terminology, I'll rename everything in a different PR.